arm-sql/2014-04-01/swagger/replicationLinks.json

```diff
@@ -0,0 +1,394 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure SQL Database API spec",
+    "description": "The Azure SQL Database management API provides a RESTful set of web services that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases.",
+    "version": "2014-04-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/replicationLinks/{linkId}": {
+      "delete": {
+        "tags": [
+          "DatabaseReplicationLinks"
+        ],
+        "operationId": "Databases_DeleteReplicationLink",
```

Comment:
>Databases_DeleteReplicationLink [](start = 24, length = 31)

For my info - Can I delete the replication link in any stage, like when data is still being replicated or pending?